src/sentry/south_migrations/0339_backfill_first_project_feature.py

```diff
@@ -0,0 +1,892 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+
+class Migration(DataMigration):
+    def forwards(self, orm):
+        db.commit_transaction()
+        try:
+            self._forwards(orm)
+        except Exception:
+            # Explicitly resume the transaction because
+            # South is going to try and roll it back, but when
+            # it can't find one, it'll error itself, masking
+            # the actual exception being raised
+            #
+            # See https://github.com/getsentry/sentry/issues/5035
+            db.start_transaction()
+            raise
+        db.start_transaction()
+
+    def _forwards(self, orm):
+        from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+        # Migrating first project, first event, and member invite
+        FEATURE_MAP = {1: 60, 2: 40, 3: 61}
+
+        OrganizationOnboardingTask = orm['sentry.OrganizationOnboardingTask']
+        FeatureAdoption = orm['sentry.FeatureAdoption']
+
+        queryset = OrganizationOnboardingTask.objects.filter(task__in=[1, 2, 3], status=1)
+        for oo_task in RangeQuerySetWrapperWithProgressBar(queryset):
+            try:
+                with transaction.atomic():
+                    feature_id = FEATURE_MAP[oo_task.task]
+                    FeatureAdoption.objects.create(
+                        feature_id=feature_id,
+                        organization_id=oo_task.organization_id,
+                        complete=True,
+                    )
+                    FeatureAdoption.objects.set_cache(
```

Comment:
as an aside, it'd probably be good to make it so that one of your organizations locally will actually be affected by this and hit this path so you can make sure the migration will actually run. this would hard error.